README.md

```diff
@@ -35,6 +35,12 @@ Available imports:
 import fetch, { Headers, Request, Response, AbortController } from 'fetch';
 ```
 
+### Use with TypeScript
+To use `ember-fetch` with TypeScript or enable editor's type support, add `"fetch": "ember-cli/ember-fetch"` to your app's `devDependencies`.
+This will get the current state of `ember-fetch` from this GitHub repo as a dependency.
```

Comment:
I'm not sure we want to advise that people do this, since they'll end up tracking the master branch rather than SemVer versioned releases.

Reply:
I tend to document `tsconfig.json` only.

Reply:
please clarify. This pertains to `package.json`, not `tsconfig.json`

> add `"fetch": "ember-cli/ember-fetch"` to your app's `devDependencies`.
This will get the current state of `ember-fetch` from this GitHub repo as a dependency.

Reply:
I removed the `"fetch": "ember-cli/ember-fetch"` instruction and only left a `tsconfig` example